gtk/src/prelude.rs

```diff
@@ -4,8 +4,18 @@
 
 //! Traits and essential types intended for blanket imports.
 
+#[doc(hidden)]
+pub use atk::prelude::*;
```

Comment:
Just wondering: do we really want to reexport every dependencies' prelude?

Reply:
Especially the preludes, yes.